# Tables of constants and thermodynamic parameters

<p><i>R</i> = 8.314 J mol⁻¹ K⁻¹<br><i>M</i><sub><small>r</small></sub> (CO₂) = 44.01 g mol⁻¹</p>
<table style="margin-left:0;margin-right:auto">
    <tr>
        <th><br>material</th>
        <th style="text-align:center">∆<sub><small>f</small></sub><i>H</i>° (298 K)<br>(J mol⁻¹)</th>
        <th style="text-align:center">C<sub><small>p</small></sub> (298 K)<br>(J mol⁻¹ K⁻¹)</th>
        <th style="text-align:center">C<sub><small>p</small></sub> (298 K)*<br>(J mol⁻¹ K⁻¹)</th>
    </tr>
    <tr>
        <td>C₂H₆</td>
        <td>–393.51</td>
        <td>52.63</td>
        <td>14.73 + 0.1272<i>T</i></td>
    </tr>
</table>
<p><sup><small>* <i>T</i> is absolute temperature in K</small></sup></p>

In [1]:
import numpy as np # be sure to run this line first!

# Definitions and short answers

### (a1) Define *heat* in a complete sentence.

Heat is a form of energy that is transferred between a system and its surrounding as a result of temperature difference. Heat is represneted by q and measured is Joules(J) which is equal to $q=C\Delta T$ where C is Heat capacity and Delta T is Temperature.

### (a2) If the system is a hot piece of metal that’s just been removed from an oven, what is the sign on *q*?

The hot metal will cool off, so it’s losing heat and *q* < 0. Therefore the sign is negative.

### (b1) Define *work of expansion* in a complete sentence. 

Expansion work is the energy transferred from a system when the volume increases against an external pressure.

### (b2) If the system is a sealed bottle of a carbonated drink (like beer, soda or cider) and you open it, what is the sign on *w*?

The system has done work and lost energy, so *w* < 0 and the sign is negative again.

### (c1) Define *internal energy* in a complete sentence.

Internal energy (*U*) is the all the energy contained in a thermodynamic system except for the energy (work) required to make space for its existence. Internal energy consists of potential energy (energy stored in a lattice and in chemical bonds, mainly) and kinetic energy (energy stored in vibrations, rotations and translation).

### (c2) How does internal energy relate to heat and work for a reversible process?

∆*U* = *q* + *w* or (better) d*U* = d*q* + d*w*

### (d1) List the four requirements for a gas to be ideal.

1. Point-like particles(with a finite mass)
2. Constant motion
3. Random motion
4. Non-interacting

OR

<br>
    There are essentially five points covered in there, but they're often grouped. Full marks requires all these aspects to be noted:
    <ul>
        <li>Particles have mass</li>
        <li>Particles have no size</li>
        <li>The motion is constant and random</li>
        <li>There are no bonding interactions, only elastic collisions</li>
    </ul>
    There are other possible aspects that might appear related to number of particles, size of the container, their speed/average energy, and more. But the points above are the ones that count toward the mark for this question.

### (d2) How is internal energy stored in an ideal gas?

Energy is stored in translational motion only.

### (d3) Which state variable fully determines the stored energy of an ideal gas?

The amount of energy stored is a function of absolute temperature (*T*) only.
*U* = 3/2*k*<sub><small>B</small></sub>*T* (per particle) or 3/2*RT* (per mole).

### (e1) Define *adiabatic process* in a complete sentence.

The process in which pressure, volume and temperature of a system change in such a manner that during the change no heat enters or leaves the system is called adiabatic process. Thus in adiabatic process, the total heat of the system remains constant.

OR

An adiabatic process is one in which no heat is exchanged between the system and the surroundings. That is, it is well insulated or *q* = 0.

### (e2) Give the equations internal energy change in terms of work for an adiabatic process.

  We start from ∆<i>U</i> = <i>q</i> + <i>w</i>, which is a statement of the first law of thermodynamics. For an adiabatic process, <i>q</i> = 0. So, the equation simplifies to the answer.
  
$\Delta U = w$

### (e3) Give a more specific equation for ∆*U* if the system is reversible and is an ideal gas.

We know, $\Delta U = q + W$

Also for an reversible process its being at equilibrium so
And reversible work is isotherm

So,

$W=-\int_{V_{i}}^{V_{f}} p_{ext}dV$ 

$pv=nrT$

Then, $p=\frac{nrT}{V}$

$W=-nrT\int_{V_{i}}^{V_{f}}\frac{dV}{V}$

$W= -nrT \ln \left ( \frac{V_{f}}{V_{i}} \right )$ which is the desired equation.


In simple words, $\Delta U = -\int_{V_{\textrm{i}}}^{V_{\textrm{f}}}p \textrm{d}V = -nRT \textrm{ln} \left ( \frac{V_{\textrm{f}}}{V_{\textrm{i}}} \right )$

### (f1) Define *isothermal process* in a complete sentence.

The process in which change in pressure and volume takes place at a constant temperature, is called a isothermal change.
In this change, the total amount of heat of the system doesn't remain constant.

### (f2) Give the value of ∆*T* for an isothermal process involving an ideal gas.

$ \Delta T=0$

Temperature is a state variable, so ∆<i>T</i> = 0 means that the temperature is the same at the start and the end of a process, but the temperature might have changed <em>during</em> the process. This doesn't have an effect on state variables, but would effect the amount of heat or work exchanged by a system.

### (f3) Give the value of ∆*U* for an isothermal process involving an ideal gas.

$\Delta U= 0$

In (d3), we defined the internal energy for an ideal gas as being proportional only to temperature. Therefore, the internal energy change will only depend on temperature change: ∆<i>U</i> = (3/2) <i>k</i><sub><small>B</small></sub> ∆<i>T</i>. And ∆<i>T</i> = 0, so ∆<i>U</i> = 0.

### (f4) Give equation for *q* in terms of *w* for an isothermal process.

$q= -W$

The answer needs to look like this to get 1 mark: <i>q</i> on the left side with nothing else, –<i>w</i> on the right side. No marks if the minus sign is missing, 0.5 marks if it's written as <i>q</i> + <i>w</i> = 0.

### (g1) Define *enthalpy* in a complete sentence that is more than just putting the equation in words.

Enthalpy is the total sum of the internal energy and the product of the pressure and volume of a thermodynamic system. So, its just another way to quantify energy. 
The equation is:

$H= U+pV$   

Where H is Enthalpy, U is Internal energy, p is Pressure and V is Volume.

### (g2) Give the equation for an infinitessimal change in enthalpy (d*H*) in terms of internal energy, pressure and volume. 

$dH= dU+ pdV + Vdp$ 

### (h1) Define *molar heat capacity* in a complete sentence.

Molar heat capacity (*C*) is the amount of energy needed to increase the temperature of one mole of a substance by one degree of temperature.

### (h2) Write the equation for the most general form of heat capacity, *C*, using derivative notation.

$C=\frac{\partial q}{\partial T}$

### (h3) Write the matching equation for constant-volume heat capacity, *C*<sub><small><i>V</i></small></sub>, using derivative notation.

$C_{V}= \left ( \frac{\partial U}{\partial T} \right )_{V}$

### (h4) Write the matching equation for constant-pressure heat capacity, *C*<sub><small><i>p</i></small></sub>, using derivative notation.

$C_{P}= \left ( \frac{\partial H}{\partial T} \right )_{P}$

# Gas expansion and work

## A 5.0 g block of solid carbon dioxide (‘dry ice’) is allowed to sublimate inside a sealed piston that total has an initial volume 1.0 × 10² cm³ and is maintained at 20 °C. At the start, the pressure of the gas inside and out is equal.

### (a) Calculate the work done when the system expands isothermally against a pressure of 1.0 bar. State any assumptions.

External pressure is fixed (that is, *p*<sub><small>int</small></sub> ≠ *p*<sub><small>ext</small></sub>), so this is irreversible expansion.

**Assumptions**  
Assume gas behaves ideally.  
Assume volume of dry ice is negligible.

In [6]:
# basic parameters
temp = 20 + 273.15 # temperature in K (0 dp, will be 3 sf)
V_i = 1.0e2 * 1e-6 # initial volume in m³ (2 sf)
p_ext = 1.0e5 # external pressure in Pa (2 sf)
moles = 5.0 / 44.01 # number of moles of CO₂ (2 sf)
R = 8.314 # gas constant, exact
delta_V = moles * R * temp / p_ext # change in volume (2 sf)

$w = -\int _{V_{\textrm{i}}}^{V_{\textrm{f}}} p_{\textrm{ext}}\textrm{d}V = -p_{\textrm{ext}} \Delta V$

In [7]:
w_irr = -p_ext * delta_V
print(w_irr) # needs to be rounded to 2 sf

-276.89719381958645


In [8]:
print('Irreversible work: {:.2f} kJ'.format(w_irr/1000))

Irreversible work: -0.28 kJ


### (b) Calculate the work done when the system expands isothermally and reversibly to the same final volume.

Reversible, so *p*<sub><small>int</small></sub> ≠ *p*<sub><small>ext</small></sub> = *p*, which can't be taken out of the integral.

$w = -\int _{V_{\textrm{i}}}^{V_{\textrm{f}}} p\textrm{d}V = -\int _{V_{\textrm{i}}}^{V_{\textrm{f}}} nRT\textrm{d}V = -nRT \int _{V_{\textrm{i}}}^{V_{\textrm{f}}}\textrm{d}V = -nRT \left (\textrm{ln} V_{\textrm{f}}-\textrm{ln}V_{\textrm{i}}\right ) = -nRT\textrm{ ln}\frac{ V_{\textrm{f}}}{V_{\textrm{i}}}$

In [11]:
V_f = V_i + delta_V
w_rev = -moles * R * temp * np.log(V_f / V_i) # note that log in numpy is base e
print(w_rev) # need to round to 2 sf
print('Reversible work: {:.2f} kJ'.format(w_rev/1000))

-929.4161882083047
Reversible work: -0.93 kJ


# Heat capacity and temperature changes

## *Assume a constant pressure of 1 bar.*

### (a) Use the data in the tables at the top to calculate the heat required to raise the temperature of ethane (C₂H₆) from 298 K to 450 K, *assuming the heat capacity is constant with temperature*.

Constant pressure, so *q* = ∆*H*

$q = \Delta _{\textrm{f}}H^{\circ}(\textrm{450 K}) - \Delta _{\textrm{f}}H^{\circ}(\textrm{298 K})=\int _{\textrm{298 K}}^{\textrm{450 K}} C_{p}\textrm{d}T = C_{p} (\textrm{450 K} - \textrm{298 K})$

In [13]:
a = 52.63*(450-298) # Calculation in Python, explanation in text
print(a)

7999.76


$q = C_{p} \Delta T = 52.63 · (450–298) = 8.00 \textrm{ kJ mol}^{-1}$

### (b) A more accurate value of the molar heat capacity of ethane is given on the table at the back, and has the form *C<sub>p</sub>* = *a* + *bT*, where the *C<sub>p</sub>* is in J mol⁻¹ K⁻¹, *T* is in kelvin and *a* and *b* are empirically derived constants. Use this value for the heat capacity to calculate the heat required for the same temperature change. *Be careful with your integration!*

$q = \int _{\textrm{298 K}}^{\textrm{450 K}} C_{p}\textrm{d}T = 14.73\left [T \right ] _{\textrm{298 K}}^{\textrm{450 K}} + \frac{0.1272}{2}\left [ T^{2} \right ] _{\textrm{298 K}}^{\textrm{450 K}}$

In [16]:
b1 = 14.73 * (450 - 298) / 1000 # limits to 3 sf, converted into kJ
print(b1)

2.23896


In [17]:
b2 = 0.1272 / 2 * (450 ** 2 - 298 ** 2) / 1000 # limits to 3 sf, converted into kJ
print(b2)

7.231065600000001


In [18]:
print(b1 + b2) # will need to be rounded to 2 dp

9.470025600000001


9.47 kJ mol⁻¹

### (c) Calculate the relative (percent) error in (a), assuming that the answer in (b) is more accurate. Use a positive sign if (a) is an overestimate and a negative sign if it is an underestimate.

In [19]:
rel_err = (a / 1000 - (b1 + b2)) / (b1 + b2)
# difference in the numerator has 2 dp (and 3 sf); denominator has 3 sf, so the quotient will have 3 sf
print('Relative uncertainty in (a): {:.1f}%'.format(rel_err*100))

Relative uncertainty in (a): -15.5%
